Doxyfile.in

```diff
@@ -0,0 +1,1552 @@
+# Doxyfile 1.6.2
```

Comment:
"'I'd argue that eventually we'd want to get rid of the Makefile build
system all together"

That will not be happening unless we do something like Bazel support.

-Z-


On Thu, Jun 18, 2015 at 11:25 AM, Dan Liew <notifications@github.com> wrote:

> In Doxyfile.in
> <https://github.com/halide/Halide/pull/825#discussion_r32761840>:
>
> > @@ -0,0 +1,1552 @@
> > +# Doxyfile 1.6.2
>
> It is a new copy :(
>
> I really didn't want to do that but it is essential for out of source
> builds to work (if you take a look in Doxyfile.in you'll see there are a
> bunch of @ substitutions in there).
>
> I'd rather there wasn't two. I'd argue that eventually we'd want to get
> rid of the Makefile build system all together, when we can drop having
> two Doxyfiles.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/825/files#r32761840>.
>